src/common/prop.c

```diff
@@ -501,6 +515,15 @@ daos_prop_entry_copy(struct daos_prop_entry *entry,
 			return rc;
 		}
 		break;
+	case DAOS_PROP_PO_POLICY:
+		D_ASSERT(entry->dpe_val_ptr != NULL);
+		D_ALLOC(entry_dup->dpe_val_ptr, sizeof(*entry->dpe_val_ptr));
```

Comment:
dpe_val_ptr is a void pointer, I think you want to allocate actual size buffer here but not the sizeof(*entry->dpe_val_ptr), right?

Reply:
Absolutely right. Fixed.